buildSrc/src/main/kotlin/io/spine/internal/gradle/ProjectExtensions.kt

```diff
@@ -72,16 +73,22 @@ fun <T : Task> Project.findTask(name: String): T {
 }
 
 /**
- * Obtains the Maven artifact ID of the project taking into account
- * the value of the [PublishExtension.spinePrefix] property.
+ * Obtains Maven artifact ID of this [Project].
  *
- * If the project has a [PublishExtension] installed, then the extension is used for
- * [obtaining][PublishExtension.artifactId] the artifact ID.
+ * The method checks if [SpinePublishing] extension is configured upon this project. If yes,
```

Comment:
What happens otherwise?

Reply:
Placed `if` and `otherwise` clauses into the same paragraph.